bot/extensions/wikipedia_cog.py

```diff
@@ -0,0 +1,85 @@
+from discord.ext.commands import Cog, hybrid_command
+import discord
+import urllib.request
+import json
+
+
+def search_results(search):
+        urlencode = urllib.parse.quote(search)
```

Comment:
I would recommend avoiding writing your multiple words variable name that way. It is harder to read what is written. `url_encoded` would be a better name (and by convention, underscore is preferred).